.github/workflows/go.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Build executable
         run: make
 
-      - name: Run attestation command integration Tests
+      - name: Run attestation command set integration tests
+        shell: bash
         run: |
-          ./test/integration/attestation-cmd/download-and-verify-package-attestation.sh
-          ./test/integration/attestation-cmd/verify-sigstore-bundle-versions.sh
+          ./test/integration/attestation-cmd/run-all-tests.sh "${{ matrix.os }}"
```

Comment:
@williammartin since you got tagged in this, I have added a helper script for the `gh attestation` command set integration tests. It takes the `matrix.os` value as an input since this bug fix is specific to the Windows platform. The new `gh attestation download` integration test will check whether the matrix.os value is `windows-latest` and check for a Windows specific value if so.